wsl-agent.md

```
@@ -0,0 +1,160 @@
+```
+
+To specify your own configuration files:
+
+```shell
+TestStream.Runner -d path_to\runner-config.json -h path_to\config\configuration.json
+```
+
+The application will bind and attach the devices into WSL, run the docker container with the proper settings. In case a device is deattached, it will automatically reattach it.
+
+> [!Important]
+> In the current state of the application, no restart will be provided in case the container stops or WSL stops.
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)